cmd/update-rules/README.md

```diff
@@ -50,7 +50,7 @@ Run the command line as:
 ```
   update-rules -h
 
-  Usage: update-rules --branch BRANCH --rules PATHorURL [--go VERSION | -o PATH]
+  Usage: update-rules --branch BRANCH --rules PATHorURL [--go VERSION | -o PATH | -remove true/false]
```

Comment:
Is remove or delete the better flag to use?

Reply:
Thinking out loud. Just like how we remove a branch in `git`. A `-d` or `--delete`